*** ./smali/android/preference/PreferenceFrameLayout$LayoutParams.smali	2014-08-20 14:01:35.600469414 +0200
--- /home/Gava97/patchrom/kumquat/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceFrameLayout$LayoutParams.smali	2014-08-20 14:00:31.204468572 +0200
***************
*** 59,64 ****
--- 59,72 ----
  
      iput-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
  
+     iget-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
+ 
+     invoke-static {p1, v1}, Landroid/preference/Injector$PreferenceFrameLayoutHook;->isRemoveBorders(Landroid/content/Context;Z)Z
+ 
+     move-result v1
+ 
+     iput-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
+ 
      invoke-virtual {v0}, Landroid/content/res/TypedArray;->recycle()V
  
      return-void
