*** ./smali/android/hardware/SensorManager.smali	2014-08-20 14:01:27.952469314 +0200
--- /home/Gava97/patchrom/kumquat/temp/new_smali/framework.jar.out/./smali/android/hardware/SensorManager.smali	2014-08-20 14:00:23.780468475 +0200
***************
*** 229,234 ****
--- 229,238 ----
      .parameter "p"
  
      .prologue
+     invoke-static {p0}, Landroid/hardware/Injector$SensorManagerHook;->getSeaLevelPressure(F)F
+ 
+     move-result p0
+ 
      const v0, 0x3e42dcae
  
      .local v0, coef:F
***************
*** 2332,2337 ****
--- 2336,2351 ----
  
  
  # virtual methods
+ .method public calibrateSensor(Landroid/hardware/Sensor;)Z
+     .locals 1
+     .parameter "sensor"
+ 
+     .prologue
+     const/4 v0, 0x0
+ 
+     return v0
+ .end method
+ 
  .method public cancelTriggerSensor(Landroid/hardware/TriggerEventListener;Landroid/hardware/Sensor;)Z
      .locals 1
      .parameter "listener"
