*** ./smali/android/os/PowerManager.smali	2014-08-20 14:01:50.796469613 +0200
--- /home/Gava97/patchrom/kumquat/temp/new_smali/framework.jar.out/./smali/android/os/PowerManager.smali	2014-08-20 14:00:44.316468743 +0200
***************
*** 14,19 ****
--- 14,21 ----
  # static fields
  .field public static final ACQUIRE_CAUSES_WAKEUP:I = 0x10000000
  
+ .field public static final BRIGHTNESS_DIM:I = 0x14
+ 
  .field public static final BRIGHTNESS_OFF:I = 0x0
  
  .field public static final BRIGHTNESS_ON:I = 0xff
