*** ./smali/android/net/wifi/WifiMonitor.smali	2014-08-20 14:01:33.572469388 +0200
--- /home/Gava97/patchrom/kumquat/temp/new_smali/framework.jar.out/./smali/android/net/wifi/WifiMonitor.smali	2014-08-20 14:00:29.540468550 +0200
***************
*** 180,185 ****
--- 180,187 ----
  
  .field private static final UNKNOWN:I = 0xa
  
+ .field private static final WAPI_EVENT_PREFIX_STR:Ljava/lang/String; = "WAPI:"
+ 
  .field private static final WPA_EVENT_PREFIX_STR:Ljava/lang/String; = "WPA:"
  
  .field private static final WPA_RECV_ERROR_STR:Ljava/lang/String; = "recv error"
